src/component/parser/json/converter/EventDefinitionConverter.ts

```diff
@@ -14,30 +14,20 @@
  * limitations under the License.
  */
 import { TDefinitions } from '../../../../model/bpmn/json/BPMN20';
-import { ShapeBpmnEventKind } from '../../../../model/bpmn/internal/shape';
+import { bpmnEventKinds } from '../../../../model/bpmn/internal/shape';
 import { TEventDefinition } from '../../../../model/bpmn/json/baseElement/rootElement/eventDefinition';
-import { ensureIsArray } from './ConverterUtil';
-
-export const bpmnEventKinds = Object.values(ShapeBpmnEventKind).filter(kind => {
-  return kind != ShapeBpmnEventKind.NONE;
-});
-
-const eventDefinitionsOfDefinitions: Map<string, ShapeBpmnEventKind> = new Map();
-
-export function findEventDefinitionOfDefinitions(id: string): ShapeBpmnEventKind {
-  return eventDefinitionsOfDefinitions.get(id);
-}
+import { ConvertedElements, ensureIsArray } from './utils';
 
 export default class EventDefinitionConverter {
+  constructor(readonly convertedElements: ConvertedElements) {}
+
   deserialize(definitions: TDefinitions): void {
     try {
-      eventDefinitionsOfDefinitions.clear();
```

Comment:
We have to clean the maps before each new parsing, because it's keep the old converted elements of the previous file.
Some elements can be in conflict if there are the same id.

Reply:
Forgot my comment. I didn't see that we instantiate a new BPMNParser (and new ConvertedElements) at each loading ^^